src/styles/sidebar-tutorial.scss

```diff
@@ -44,10 +54,56 @@
   color: $gray-light;
 }
 
+.sidebar-tutorial__list-item--is-theme-clean {
+  font-size: 14px;
+}
+
+.sidebar-tutorial__list-item--is-theme-clean::before {
+  color: $gray-dark;
+}
+
 .sidebar-tutorial__list-item-content {
   margin-top: 1em; /* Put vertical space in-between list items, equal to the
                       space between normal paragraphs.
                       Note: This also puts the same amount of space above the
                       first list item (i.e. between the list and whatever's
                       above it). */
 }
+
+.sidebar-tutorial__list-item-annotate {
+  background-color: $highlight-color-focus;
+  padding: 0px 3px;
+}
+
+.sidebar-tutorial__list-item-new-note-btn {
+  background-color: #626262;
```

Comment:
This should use the `$color-dove-gray` variable for consistency with other elements that are styled using the same palette element from the color palette.